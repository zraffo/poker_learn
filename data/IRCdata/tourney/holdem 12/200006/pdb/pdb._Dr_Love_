_Dr_Love_ 960406544 15  6 r   krA   -     -         7000 7000    0 Ks Kd
_Dr_Love_ 960413953 10  3 f   -     -     -        10000    0    0 
_Dr_Love_ 960414013 10  2 Bk  kr    -     -        10000 5051 5251 
_Dr_Love_ 960414123 10  1 Bc  k     kc    b        10200 5251 5601 
_Dr_Love_ 960414257 10 10 c   k     c     f        10550  400    0 
_Dr_Love_ 960414310 10  9 f   -     -     -        10150    0    0 
_Dr_Love_ 960414374 10  8 f   -     -     -        10150    0    0 
_Dr_Love_ 960414416 10  7 cc  r     b     -        10150 10149 24351 
_Dr_Love_ 960414513 10  6 c   kc    b     b        24352 10294 14193 7h Kc
_Dr_Love_ 960414616  9  4 c   kf    -     -        28251  200    0 
_Dr_Love_ 960414686  9  4 cc  kc    k     b        28051 4351 2001 9d Kc
_Dr_Love_ 960414738  9  3 f   -     -     -        25701    0    0 
_Dr_Love_ 960414772  9  2 Bc  b     b     b        25701 14703 13907 Qc 2c
_Dr_Love_ 960414813  9  1 Bf  -     -     -        24905  100    0 
_Dr_Love_ 960414850  9  9 f   -     -     -        24805    0    0 
_Dr_Love_ 960414859  9  8 c   k     kc    kr       24805 9200 2650 9s 8d
_Dr_Love_ 960414938  9  7 f   -     -     -        18255    0    0 
_Dr_Love_ 960414962  8  5 f   -     -     -        18255    0    0 
_Dr_Love_ 960414996  8  5 f   -     -     -        18255    0    0 
_Dr_Love_ 960415023  8  4 c   k     k     k        18255  400    0 5d Kd
_Dr_Love_ 960415053  8  3 c   kc    b     k        17855 4801    0 9h Kh
_Dr_Love_ 960415130  8  2 Bk  k     kf    -        13054  400    0 
_Dr_Love_ 960415193  7  1 Bc  kr    bA    -        12654 12654 13054 2c 3h
_Dr_Love_ 960415256  7  7 f   -     -     -        13054    0    0 
_Dr_Love_ 960415297  7  6 f   -     -     -        13054    0    0 
_Dr_Love_ 960415306  7  5 f   -     -     -        13054    0    0 
_Dr_Love_ 960415322  6  4 c   k     k     b        13054 5801 12802 Qh Th
_Dr_Love_ 960415359  6  3 f   -     -     -        20055    0    0 
_Dr_Love_ 960415376  6  2 Bk  b     bc    -        20055 10925 27052 Jc 4d
_Dr_Love_ 960415483  5  1 Bc  k     k     k        36182  800 1600 8h Kc
_Dr_Love_ 960415512  5  5 c   rc    -     -        36982 8145 17090 5s 7s
_Dr_Love_ 960415543  4  3 c   bc    bc    -        45927 38348    0 Tc 9c
_Dr_Love_ 960415580  4  3 f   -     -     -         7579    0    0 
_Dr_Love_ 960415597  4  2 B   -     -     -         7579 1600 2400 
_Dr_Love_ 960415601  4  1 Bf  -     -     -         8379  800    0 
_Dr_Love_ 960415609  4  4 f   -     -     -         7579    0    0 
_Dr_Love_ 960415620  4  3 f   -     -     -         7579    0    0 
_Dr_Love_ 960415628  4  2 B   -     -     -         7579 1600 2400 
_Dr_Love_ 960415631  4  1 Bc  k     b     -         8379 4800 6400 
_Dr_Love_ 960415645  4  4 f   -     -     -         9979    0    0 
_Dr_Love_ 960415653  4  3 c   b     -     -         9979 9978 12378 
_Dr_Love_ 960415668  4  2 B   -     -     -        12379 1600 2400 
_Dr_Love_ 960415673  4  1 Bc  k     k     b        13179 13178 14778 
_Dr_Love_ 960415702  4  4 f   -     -     -        14779    0    0 
_Dr_Love_ 960415710  4  3 f   -     -     -        14779    0    0 
_Dr_Love_ 960415717  4  2 B   -     -     -        14779 1600 2400 
_Dr_Love_ 960415726  4  1 Bf  -     -     -        15579  800    0 
_Dr_Love_ 960415729  4  4 c   k     k     k        14779 1600    0 7d 2d
_Dr_Love_ 960415747  4  3 c   -     -     -        13179 1600 2850 Ad Js
_Dr_Love_ 960415754  3  1 Bk  kc    krA   -        14429 14429    0 3h Ac
_Dr_Love_ 960422973 23 18 rcA -     -     -        10000 10000 30676 3c 3d
_Dr_Love_ 960423063 21 15 f   -     -     -        30676    0    0 
_Dr_Love_ 960423251 21 14 f   -     -     -        30676    0    0 
_Dr_Love_ 960423337 21 13 f   -     -     -        30676    0    0 
_Dr_Love_ 960423433 21 12 r   f     -     -        30676 2001    0 
_Dr_Love_ 960423547 19 10 f   -     -     -        28675    0    0 
_Dr_Love_ 960423610 17  8 f   -     -     -        28675    0    0 
_Dr_Love_ 960423702 17  7 f   -     -     -        28675    0    0 
_Dr_Love_ 960423774 17  6 cc  b     -     -        28675 2551 5351 
_Dr_Love_ 960423847 17  5 cf  -     -     -        31475  200    0 
_Dr_Love_ 960423924 17  4 cc  b     k     c        31275 15350    0 Jd 6d
_Dr_Love_ 960424041 17  3 c   b     -     -        15925 15924 5774 Jc Th
_Dr_Love_ 960424118 17  2 Bf  -     -     -         5775  200    0 
_Dr_Love_ 960424197 16  1 Bf  -     -     -         5575  100    0 
_Dr_Love_ 960424315 16 16 f   -     -     -         5475    0    0 
_Dr_Love_ 960424382 16 15 ccA -     -     -         5475 5475    0 Ad 4h
_Dr_Love_ 960442424 15 10 c   f     -     -        10000  550    0 
_Dr_Love_ 960442558 15  9 f   -     -     -         9450    0    0 
_Dr_Love_ 960442626 15  8 f   -     -     -         9450    0    0 
_Dr_Love_ 960442725 15  7 f   -     -     -         9450    0    0 
_Dr_Love_ 960442793 15  6 f   -     -     -         9450    0    0 
_Dr_Love_ 960442885 15  5 cc  rA    -     -         9450 9450 15650 Kh 8d
_Dr_Love_ 960442952 14  4 f   -     -     -        15650    0    0 
_Dr_Love_ 960443022 14  3 f   -     -     -        15650    0    0 
_Dr_Love_ 960443060 14  2 Bk  kc    k     b        15650 9869 14319 
_Dr_Love_ 960443157 14  1 Bf  -     -     -        20100   50    0 
_Dr_Love_ 960443230 14 14 c   k     k     f        20050  100    0 
_Dr_Love_ 960443334 14 13 r   k     b     -        19950 4002 6903 
_Dr_Love_ 960443416 14 12 f   -     -     -        22851    0    0 
_Dr_Love_ 960443470 13 10 c   b     -     -        22851 2201 2901 
_Dr_Love_ 960443525 13  9 f   -     -     -        23551    0    0 
_Dr_Love_ 960443562 13  8 f   -     -     -        23551    0    0 
_Dr_Love_ 960443655 13  7 rr  -     -     -        23551 21302 51453 Qh Qd
_Dr_Love_ 960443736 11  5 f   -     -     -        53702    0    0 
_Dr_Love_ 960443789 11  4 r   bA    -     -        53702 53702 44377 Qd Qs
_Dr_Love_ 960443856 11  3 cc  kr    -     -        44377 13000 14000 6c Ac
_Dr_Love_ 960443914 11  2 Bk  kf    -     -        45377  400    0 
_Dr_Love_ 960443981 11  1 Br  b     b     -        44977 9203  128 2h Ac
_Dr_Love_ 960444051 11 11 c   k     k     k        35902  400    0 Td Kh
_Dr_Love_ 960444108 11 10 f   -     -     -        35502    0    0 
_Dr_Love_ 960444177 11  9 c   rA    -     -        35502 35502 58527 5c Kd
_Dr_Love_ 960444244 10  7 cc  f     -     -        58527 4400    0 
_Dr_Love_ 960444306 10  6 f   -     -     -        54127    0    0 
_Dr_Love_ 960444387 10  5 f   -     -     -        54127    0    0 
_Dr_Love_ 960444408  9  4 f   -     -     -        54127    0    0 
_Dr_Love_ 960444460  9  3 crc -     -     -        54127 21098    0 As Qd
_Dr_Love_ 960444508  9  2 Bk  k     b     -        33029 1801 2601 
_Dr_Love_ 960444581  9  1 Bc  b     -     -        33829 6601 6876 Qd Td
_Dr_Love_ 960444625  9  9 f   -     -     -        34104    0    0 
_Dr_Love_ 960444681  9  8 c   rA    -     -        34104 34104 51454 Qh Kd
_Dr_Love_ 960444745  7  6 c   k     k     k        51454 1600    0 Ts Kc
_Dr_Love_ 960444791  6  5 f   -     -     -        49854    0    0 
_Dr_Love_ 960444820  6  4 f   -     -     -        49854    0    0 
_Dr_Love_ 960444834  6  3 c   f     -     -        49854 1600    0 
_Dr_Love_ 960444863  6  2 Bk  kf    -     -        48254 1600    0 
_Dr_Love_ 960444898  6  1 Bc  b     -     -        46654 3601 5201 
_Dr_Love_ 960444931  6  6 f   -     -     -        48254    0    0 
_Dr_Love_ 960444959  4  3 f   -     -     -        48254    0    0 
_Dr_Love_ 960444999  4  2 B   -     -     -        48254 3200 4800 
_Dr_Love_ 960445016  4  1 Bc  k     k     k        49854 4150    0 8h 7s
_Dr_Love_ 960445054  4  4 c   b     k     bA       45704 45704 60505 
_Dr_Love_ 960445107  4  3 c   k     k     k        60505 3200 9600 6h 7h
_Dr_Love_ 960445139  4  2 Bk  k     k     r        66905 36400 23000 3h 2h
_Dr_Love_ 960445183  4  1 Bc  k     k     bA       53505 53505 23459 6d Kh
_Dr_Love_ 960445224  4  4 f   -     -     -        23459    0    0 
_Dr_Love_ 960445243  4  3 c   bA    -     -        23459 23459 51718 Qh 3h
_Dr_Love_ 960445274  4  2 Bk  kf    -     -        51718 3200    0 
_Dr_Love_ 960445305  4  1 Bc  k     b     -        48518 23201 26401 
_Dr_Love_ 960445336  4  4 r   -     -     -        51718 10001 18001 
_Dr_Love_ 960445351  4  3 f   -     -     -        59718    0    0 
_Dr_Love_ 960445360  4  2 Bc  bA    -     -        59718 59718 32918 2d Ac
_Dr_Love_ 960445398  4  1 Bf  -     -     -        32918 3200    0 
_Dr_Love_ 960445429  4  4 f   -     -     -        29718    0    0 
_Dr_Love_ 960445445  4  3 f   -     -     -        29718    0    0 
_Dr_Love_ 960445456  4  2 Bk  f     -     -        29718 6400    0 
_Dr_Love_ 960445473  4  1 Bc  k     k     bA       23318 23318 29718 
_Dr_Love_ 960445498  4  4 crA -     -     -        29718 29718    0 6d Ac
_Dr_Love_ 960446156 17  4 rc  bA    -     -        10000 10000    0 Qs 8s
