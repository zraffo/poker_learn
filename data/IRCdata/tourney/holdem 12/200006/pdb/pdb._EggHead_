_EggHead_ 960467979  7  6 f   -     -     -        10000    0    0 
_EggHead_ 960468013  7  5 c   k     k     b        10000 2051 2201 
_EggHead_ 960468084  7  4 f   -     -     -        10150    0    0 
_EggHead_ 960468164  7  3 cc  kf    -     -        10150  225    0 
_EggHead_ 960468226  7  2 Bc  kf    -     -         9925  175    0 
_EggHead_ 960468265  7  1 Bc  k     k     b         9750 2226    0 Tc Qh
_EggHead_ 960468303  7  7 c   k     b     -         7524 2101 2351 
_EggHead_ 960468366  7  6 f   -     -     -         7774    0    0 
_EggHead_ 960468424  7  5 f   -     -     -         7774    0    0 
_EggHead_ 960468458  7  4 cc  bA    -     -         7774 7774    0 4c 3c
