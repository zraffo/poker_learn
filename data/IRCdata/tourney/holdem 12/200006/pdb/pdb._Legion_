_Legion_  960708790 11  7 cA  -     -     -        15000 15000    0 5h Ah
