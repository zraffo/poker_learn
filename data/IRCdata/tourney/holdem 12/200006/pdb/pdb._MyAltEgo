_MyAltEgo 960227171 15  8 f   -     -     -        10000    0    0 
_MyAltEgo 960227276 15  7 f   -     -     -        10000    0    0 
_MyAltEgo 960227328 15  6 f   -     -     -        10000    0    0 
_MyAltEgo 960227424 15  5 f   -     -     -        10000    0    0 
_MyAltEgo 960227464 15  4 f   -     -     -        10000    0    0 
_MyAltEgo 960227544 15  3 r   b     -     -        10000 4002 6153 
_MyAltEgo 960227585 15  2 Bf  -     -     -        12151  100    0 
_MyAltEgo 960227614 15  1 Bc  kf    -     -        12051  100    0 
_MyAltEgo 960227679 15 15 r   b     bc    -        11951 11150    0 Js Ks
_MyAltEgo 960227803 15 14 rA  -     -     -          801  801    0 Qs Kc
_MyAltEgo 960244033 18  8 f   -     -     -        10000    0    0 
_MyAltEgo 960244145 18  7 f   -     -     -        10000    0    0 
_MyAltEgo 960244258 17  6 f   -     -     -        10000    0    0 
_MyAltEgo 960244407 17  5 f   -     -     -        10000    0    0 
_MyAltEgo 960244505 16  4 f   -     -     -        10000    0    0 
_MyAltEgo 960244550 13  3 c   f     -     -        10000  200    0 
_MyAltEgo 960244608 12  2 Bk  bcA   -     -         9800 9800    0 Ad 6d
_MyAltEgo 960263912 23 11 cc  kf    -     -        10000 1050    0 
_MyAltEgo 960264056 22 10 f   -     -     -         8950    0    0 
_MyAltEgo 960264183 22  9 f   -     -     -         8950    0    0 
_MyAltEgo 960264264 21  7 f   -     -     -         8950    0    0 
_MyAltEgo 960264389 21  6 f   -     -     -         8950    0    0 
_MyAltEgo 960264547 21  5 f   -     -     -         8950    0    0 
_MyAltEgo 960264668 21  4 f   -     -     -         8950    0    0 
_MyAltEgo 960264783 20  3 f   -     -     -         8950    0    0 
_MyAltEgo 960264873 20  2 Bc  krcA  -     -         8950 8950 20400 5h 6s
_MyAltEgo 960264982 20  1 Bf  -     -     -        20400  100    0 
_MyAltEgo 960265072 18 18 f   -     -     -        20300    0    0 
_MyAltEgo 960265191 17 16 cc  b     b     k        20300 20299    0 8s Ks
_MyAltEgo 960265358 16 15 f   -     -     -            1    0    0 
_MyAltEgo 960265479 16 14 cA  -     -     -            1    1    0 7c 9c
_MyAltEgo 960281086  8  2 Bf  -     -     -        10000   50    0 
_MyAltEgo 960281162  8  1 Bf  -     -     -         9950   25    0 
_MyAltEgo 960281221  8  8 f   -     -     -         9925    0    0 
_MyAltEgo 960281272  8  7 f   -     -     -         9925    0    0 
_MyAltEgo 960281313  8  6 f   -     -     -         9925    0    0 
_MyAltEgo 960281379  8  5 f   -     -     -         9925    0    0 
_MyAltEgo 960281468  7  4 f   -     -     -         9925    0    0 
_MyAltEgo 960281522  6  3 r   b     -     -         9925 7002 9053 
_MyAltEgo 960281559  6  2 Bk  k     f     -        11976  100    0 
_MyAltEgo 960281630  6  1 Bc  b     -     -        11876 2101 2401 
_MyAltEgo 960281662  6  6 r   b     -     -        12176 4002 6153 
_MyAltEgo 960281690  6  5 r   -     -     -        14327 2001 2151 
_MyAltEgo 960281707  6  4 r   b     -     -        14477 5002 9054 
_MyAltEgo 960281737  6  3 f   -     -     -        18529    0    0 
_MyAltEgo 960281766  5  2 Bk  c     f     -        18529  400    0 
_MyAltEgo 960281804  5  1 Br  -     -     -        18129  601  801 
_MyAltEgo 960281819  5  5 f   -     -     -        18329    0    0 
_MyAltEgo 960281856  5  4 r   bc    -     -        18329 13423 27146 Kc As
_MyAltEgo 960281889  4  2 Bk  kf    -     -        32052  200    0 
_MyAltEgo 960281928  4  1 Bcc kf    -     -        31852  600    0 
_MyAltEgo 960281957  4  4 f   -     -     -        31252    0    0 
_MyAltEgo 960281961  4  3 f   -     -     -        31252    0    0 
_MyAltEgo 960281974  4  2 Bk  kf    -     -        31252  200    0 
_MyAltEgo 960281999  4  1 Bc  b     -     -        31052 2201 2401 
_MyAltEgo 960282013  4  4 c   b     -     -        31252 5701 7301 
_MyAltEgo 960282038  4  3 f   -     -     -        32852    0    0 
_MyAltEgo 960282080  4  2 Bk  kf    -     -        32852  200    0 
_MyAltEgo 960282094  4  1 Bf  -     -     -        32652  100    0 
_MyAltEgo 960282100  4  4 f   -     -     -        32552    0    0 
_MyAltEgo 960282118  4  3 f   -     -     -        32552    0    0 
_MyAltEgo 960282134  4  2 Br  -     -     -        32552 2401 2801 
_MyAltEgo 960282147  4  1 Bc  krc   -     -        32952 13475 27350 5h Ts
_MyAltEgo 960282199  3  3 cc  f     -     -        46827 1600    0 
_MyAltEgo 960282232  3  2 Bf  -     -     -        45227  400    0 
_MyAltEgo 960282240  3  1 Bc  k     k     kf       44827  400    0 
_MyAltEgo 960282278  3  3 f   -     -     -        44427    0    0 
_MyAltEgo 960282290  3  2 Br  c     -     -        44427 3824 7648 7s As
_MyAltEgo 960282320  2  1 Bf  -     -     -        48251  200    0 
_MyAltEgo 960282326  2  2 Bf  -     -     -        48051  400    0 
_MyAltEgo 960282332  2  1 Bcc kr    bf    -        47651 8202    0 
_MyAltEgo 960282369  2  2 B   -     -     -        39449  400  600 
_MyAltEgo 960282372  2  1 Bf  -     -     -        39649  200    0 
_MyAltEgo 960282376  2  2 Bc  f     -     -        39449 1200    0 
_MyAltEgo 960282384  2  1 Bf  -     -     -        38249  200    0 
_MyAltEgo 960282387  2  2 Bc  rcA   -     -        38049 38049    0 Kc 9d
_MyAltEgo 960282628  7  3 cc  kf    -     -        10000  450    0 
_MyAltEgo 960282667  7  2 Bk  k     b     -         9550 2101 2201 
_MyAltEgo 960282702  7  1 Bf  -     -     -         9650   50    0 
_MyAltEgo 960282772  7  7 r   rcA   -     -         9600 9600 19350 Ac 4c
_MyAltEgo 960282834  7  6 cc  b     b     -        19350 9799 16050 
_MyAltEgo 960282952  7  5 f   -     -     -        25601    0    0 
_MyAltEgo 960282991  7  4 c   rc    k     k        25601 6651    0 Ks Ac
_MyAltEgo 960283051  6  3 f   -     -     -        18950    0    0 
_MyAltEgo 960283100  5  2 Bk  k     k     k        18950  200    0 Tc 7h
_MyAltEgo 960283164  5  1 Bc  kf    -     -        18750  700    0 
_MyAltEgo 960283209  5  5 r   b     -     -        18050 3502 4303 
_MyAltEgo 960283235  5  4 f   -     -     -        18851    0    0 
_MyAltEgo 960283296  5  3 f   -     -     -        18851    0    0 
_MyAltEgo 960283346  5  2 Bc  kf    -     -        18851 1000    0 
_MyAltEgo 960283382  5  1 Bf  -     -     -        17851  200    0 
_MyAltEgo 960283393  5  5 r   bc    -     -        17651 16952 34504 Kd Ad
_MyAltEgo 960283435  4  3 c   f     -     -        35203  400    0 
_MyAltEgo 960283473  4  2 Bf  -     -     -        34803  400    0 
_MyAltEgo 960283498  4  1 Br  -     -     -        34403 5201 2454 Th Td
_MyAltEgo 960283516  4  4 c   f     -     -        31656 1400    0 
_MyAltEgo 960283546  3  2 Bf  -     -     -        30256  400    0 
_MyAltEgo 960283561  3  1 Bc  k     b     -        29856 2401 2801 
_MyAltEgo 960283582  3  3 c   b     -     -        30256 10401 11201 
_MyAltEgo 960283618  3  2 Bk  k     k     k        31056  800 2000 7h Qh
_MyAltEgo 960283644  3  1 Bf  -     -     -        32256  400    0 
_MyAltEgo 960283675  3  3 c   f     -     -        31856  800    0 
_MyAltEgo 960283688  3  2 Bf  -     -     -        31056  800    0 
_MyAltEgo 960283709  2  1 Bf  -     -     -        30256  400    0 
_MyAltEgo 960283720  2  2 Bk  b     -     -        29856 10801 11601 
_MyAltEgo 960283734  2  1 Bf  -     -     -        30656  400    0 
_MyAltEgo 960283743  2  2 Bk  k     k     f        30256  800    0 
_MyAltEgo 960283770  2  1 Bc  kc    k     k        29456 2400    0 Ad 3h
_MyAltEgo 960283792  2  2 Bf  -     -     -        27056  800    0 
_MyAltEgo 960283802  2  1 Bc  k     kc    kf       26256 2400    0 
_MyAltEgo 960283822  2  2 B   -     -     -        23856  800 1200 
_MyAltEgo 960283827  2  1 Bf  -     -     -        24256  400    0 
_MyAltEgo 960283829  2  2 Bk  k     k     k        23856  800    0 2h Ts
_MyAltEgo 960283848  2  1 Bc  kc    kc    kf       23056 7200    0 
_MyAltEgo 960283879  2  2 Bk  k     k     k        15856  800    0 Qc Td
_MyAltEgo 960283900  2  1 Bc  k     kf    -        15056  800    0 
_MyAltEgo 960283918  2  2 B   -     -     -        14256  800 1200 
_MyAltEgo 960283921  2  1 Bc  k     k     k        14656 1600 3200 Jc 9c
_MyAltEgo 960283949  2  2 Bk  k     k     k        16256 1600    0 4h 6d
_MyAltEgo 960283974  2  1 Bf  -     -     -        14656  800    0 
_MyAltEgo 960283979  2  2 Bk  rcA   -     -        13856 13856    0 Jh Qc
_MyAltEgo 960284361  8  5 r   k     bf    -        10000 1002    0 
_MyAltEgo 960284414  8  4 r   k     r     -         8998 5010 7515 
_MyAltEgo 960284466  8  3 f   -     -     -        11503    0    0 
_MyAltEgo 960284535  8  2 Bf  -     -     -        11503  100    0 
_MyAltEgo 960284583  7  1 Br  b     -     -        11403 7052 11254 
_MyAltEgo 960284626  7  7 c   k     k     f        15605  100    0 
_MyAltEgo 960284714  7  6 cc  r     c     -        15505 9249 21298 Kd 9h
_MyAltEgo 960284779  6  4 f   -     -     -        27554    0    0 
_MyAltEgo 960284797  6  3 f   -     -     -        27554    0    0 
_MyAltEgo 960284826  6  2 Bc  k     b     -        27554 10701 12101 
_MyAltEgo 960284883  6  1 Bc  kf    -     -        28954  200    0 
_MyAltEgo 960284949  6  6 rc  c     -     -        28754 10600    0 9s As
_MyAltEgo 960285001  6  5 f   -     -     -        18154    0    0 
_MyAltEgo 960285026  6  4 c   c     r     -        18154 16400 24000 
_MyAltEgo 960285083  6  3 c   f     -     -        25754  400    0 
_MyAltEgo 960285125  6  2 Bc  kc    -     -        25354 4600    0 Qs 7s
_MyAltEgo 960285173  6  1 Bf  -     -     -        20754  200    0 
_MyAltEgo 960285196  5  5 c   k     k     r        20554 10401 21002 2s Td
_MyAltEgo 960285250  5  4 c   r     rA    -        31155 31155 14708 7h 7s
_MyAltEgo 960285293  5  3 cc  b     -     -        14708 7999 10399 
_MyAltEgo 960285390  5  2 Bc  b     -     -        17108 17107 22707 
_MyAltEgo 960285449  5  1 Bc  kf    -     -        22708 2800    0 
_MyAltEgo 960285486  5  5 c   rA    -     -        19908 19908 10710 4c 4s
_MyAltEgo 960285530  5  4 f   -     -     -        10710    0    0 
_MyAltEgo 960285548  5  3 cc  c     -     -        10710 7095 14990 3d Kd
_MyAltEgo 960285605  4  2 Br  k     k     k        18605 5801 12402 Jc As
_MyAltEgo 960285661  4  1 Bf  -     -     -        25206  400    0 
_MyAltEgo 960285690  4  4 c   k     f     -        24806  800    0 
_MyAltEgo 960285717  4  3 c   b     -     -        24006 11601 14001 
_MyAltEgo 960285747  4  2 Brc -     -     -        26406 19596 39992 7s 7d
_MyAltEgo 960285774  3  1 Bf  -     -     -        46802  800    0 
_MyAltEgo 960285798  3  3 c   f     -     -        46002 1600    0 
_MyAltEgo 960285817  2  2 Bf  -     -     -        44402 1600    0 
_MyAltEgo 960285847  2  1 Bf  -     -     -        42802  800    0 
_MyAltEgo 960285855  2  2 B   -     -     -        42002 1600 2400 
_MyAltEgo 960285858  2  1 Bcc b     -     -        42802 42801 5603 9s Kc
_MyAltEgo 960285895  2  2 Bf  -     -     -         5604 1600    0 
_MyAltEgo 960285921  2  1 Br  -     -     -         4004 4000 5600 
_MyAltEgo 960285933  2  2 B   -     -     -         5604 1600 2400 
_MyAltEgo 960285935  2  1 BccA -     -     -         6404 6404    0 Ah 4d
_MyAltEgo 960286099 10  4 f   -     -     -        10000    0    0 
_MyAltEgo 960286159  9  3 rA  -     -     -        10000 10000 10075 
_MyAltEgo 960286200  9  2 Br  k     k     kf       10075  551    0 
_MyAltEgo 960286295  9  1 Bc  k     kc    k         9524  250    0 9s 4s
_MyAltEgo 960286359  9  9 f   -     -     -         9274    0    0 
_MyAltEgo 960286424  9  8 c   f     -     -         9274  100    0 
_MyAltEgo 960286483  9  7 c   c     f     -         9174  371    0 
_MyAltEgo 960286550  9  6 c   bA    -     -         8803 8803 9103 
_MyAltEgo 960286623  9  5 c   bA    -     -         9103 9103 9453 
_MyAltEgo 960286651  9  4 f   -     -     -         9453    0    0 
_MyAltEgo 960286700  8  3 c   kc    k     krA       9453 9453 11453 
_MyAltEgo 960286781  8  2 Bf  -     -     -        11453  200    0 
_MyAltEgo 960286823  8  1 Bc  bA    -     -        11253 11253 2803 3s 4s
_MyAltEgo 960286885  8  8 f   -     -     -         2803    0    0 
_MyAltEgo 960286941  8  7 f   -     -     -         2803    0    0 
_MyAltEgo 960287009  7  5 f   -     -     -         2803    0    0 
_MyAltEgo 960287054  6  4 f   -     -     -         2803    0    0 
_MyAltEgo 960287088  6  4 f   -     -     -         2803    0    0 
_MyAltEgo 960287110  6  3 c   bA    -     -         2803 2803 3403 
_MyAltEgo 960287174  6  2 Bc  kf    -     -         3403 1000    0 
_MyAltEgo 960287218  6  1 Bf  -     -     -         2403  200    0 
_MyAltEgo 960287241  6  6 crA -     -     -         2203 2203    0 Kc 9s
