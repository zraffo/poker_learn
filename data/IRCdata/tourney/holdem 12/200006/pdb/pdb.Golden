Golden    961067828 13  6 cc  k     kf    -         5000  275    0 
Golden    961067904 13  5 f   -     -     -         4725    0    0 
Golden    961067945 13  4 f   -     -     -         4725    0    0 
Golden    961067981 13  3 f   -     -     -         4725    0    0 
Golden    961068078 13  2 Bk  kf    -     -         4725   50    0 
Golden    961068113 13  1 Bcf -     -     -         4675  225    0 
Golden    961068149 12 12 c   k     f     -         4450   50    0 
Golden    961068227 12 11 f   -     -     -         4400    0    0 
Golden    961068299 12 10 c   b     c     -         4400 4163 11226 6d 6s
Golden    961068423 11  8 c   k     f     -        11463  100    0 
Golden    961068473 11  7 f   -     -     -        11363    0    0 
Golden    961068645 11  6 f   -     -     -        11363    0    0 
Golden    961068694 11  5 f   -     -     -        11363    0    0 
Golden    961068765 11  4 f   -     -     -        11363    0    0 
Golden    961068820 11  3 f   -     -     -        11363    0    0 
Golden    961068868 11  2 Bf  -     -     -        11363  200    0 
Golden    961068880 11  1 Bf  -     -     -        11163  100    0 
Golden    961068908 11 11 f   -     -     -        11063    0    0 
Golden    961068947 11 10 f   -     -     -        11063    0    0 
Golden    961069000 11  9 f   -     -     -        11063    0    0 
Golden    961069043 10  7 f   -     -     -        11063    0    0 
Golden    961069082 10  6 f   -     -     -        11063    0    0 
Golden    961069108  9  5 f   -     -     -        11063    0    0 
Golden    961069142  9  4 f   -     -     -        11063    0    0 
Golden    961069154  9  3 f   -     -     -        11063    0    0 
Golden    961069223  9  2 Bf  -     -     -        11063  400    0 
Golden    961069246  9  1 Bc  k     k     k        10663  400    0 6s Ac
Golden    961069295  9  9 f   -     -     -        10263    0    0 
Golden    961069312  9  8 f   -     -     -        10263    0    0 
Golden    961069335  9  7 f   -     -     -        10263    0    0 
Golden    961069365  8  5 f   -     -     -        10263    0    0 
Golden    961069404  8  5 f   -     -     -        10263    0    0 
Golden    961069416  8  4 cc  c     bA    -        10263 10263    0 9d Jd
Golden    961075872  9  8 f   -     -     -         5000    0    0 
Golden    961075922  9  7 f   -     -     -         5000    0    0 
Golden    961075981  9  6 f   -     -     -         5000    0    0 
Golden    961076025  9  5 f   -     -     -         5000    0    0 
Golden    961076059  9  4 ccc kf    -     -         5000  750    0 
Golden    961076126  9  3 f   -     -     -         4250    0    0 
Golden    961076170  8  2 Bc  k     b     k         4250 1150 2775 Kd Td
Golden    961076211  8  1 Bf  -     -     -         5875   50    0 
Golden    961076242  8  8 f   -     -     -         5825    0    0 
Golden    961076284  8  7 c   b     b     k         5825 1225 2700 9d Ac
Golden    961076353  8  6 cf  -     -     -         7300  100    0 
Golden    961076410  8  5 f   -     -     -         7200    0    0 
Golden    961076459  8  4 f   -     -     -         7200    0    0 
Golden    961076498  7  2 Bk  f     -     -         7200  100    0 
Golden    961076532  7  1 Bf  -     -     -         7100  100    0 
Golden    961076580  7  7 f   -     -     -         7000    0    0 
Golden    961076591  7  6 f   -     -     -         7000    0    0 
Golden    961076616  7  5 f   -     -     -         7000    0    0 
Golden    961076634  7  4 f   -     -     -         7000    0    0 
Golden    961076643  7  3 f   -     -     -         7000    0    0 
Golden    961076666  7  2 Bc  k     k     k         7000  975    0 Ts As
Golden    961076718  6  1 Bc  kf    -     -         6025  200    0 
Golden    961076742  6  6 c   b     c     f         5825 4200    0 
Golden    961076791  6  5 f   -     -     -         1625    0    0 
Golden    961076814  6  4 f   -     -     -         1625    0    0 
Golden    961076832  6  3 rcA -     -     -         1625 1625    0 5h 5d
Golden    961077440 10  9 f   -     -     -        10000    0    0 
Golden    961077493  9  7 r   bf    -     -        10000  600    0 
Golden    961077557  9  6 f   -     -     -         9400    0    0 
Golden    961077605  9  5 c   kf    -     -         9400   50    0 
Golden    961077669  9  4 f   -     -     -         9350    0    0 
Golden    961077731  9  3 f   -     -     -         9350    0    0 
Golden    961077807  9  2 Bf  -     -     -         9350  100    0 
Golden    961077874  9  1 Bc  k     kc    kf        9250  600    0 
Golden    961077931  9  9 f   -     -     -         8650    0    0 
Golden    961077968  9  8 f   -     -     -         8650    0    0 
Golden    961078010  9  7 f   -     -     -         8650    0    0 
Golden    961078054  9  6 c   c     c     c         8650 4300    0 4h Qs
Golden    961078097  9  5 f   -     -     -         4350    0    0 
Golden    961078136  9  4 cf  -     -     -         4350  100    0 
Golden    961078178  9  3 cc  kf    -     -         4250 1300    0 
Golden    961078217  8  2 Bk  k     kf    -         2950  200    0 
Golden    961078252  8  1 Bc  kf    -     -         2750  200    0 
Golden    961078293  8  8 c   f     -     -         2550  200    0 
Golden    961078357  8  7 f   -     -     -         2350    0    0 
Golden    961078409  8  6 c   c     cA    -         2350 2350    0 7s As
